db/init.sh

```diff
@@ -1,26 +1,34 @@
+#!/usr/bin/env bash
 set -e
+
+DB_INIT_DIR=${1:-docker-entrypoint-initdb.d}
```

Comment:
It may be even better to *require* this arg to be passed and error with a helpful message if it isn't. That way it's more clear what the problem with any particular platform's package is.

Reply:
We're using this mostly for Docker, and don't really want to mess with the docker config right now. Docker automatically runs init scripts from that directory. Added an output message about the directory.